versions.txt

```diff
@@ -1,2 +1,4 @@
 # List of packages to bump:
 # Format: [package_name]@[new_version]
+
+snooker@8.0.0
```

Comment:
Was this actually a breaking change in the end? As looking at the code all we did was make it so it'll clone the table within the `CopySelected` function instead.